src/main.rs

```diff
@@ -85,11 +93,16 @@ enum Commands {
         #[clap(long)]
         auth_token: Option<String>,
         /// If this path is provided and it exists, the private key is read from this file and used, if it does not exist the private key will be persisted to this location.
+        ///
+        /// If this path is not provided and persistent is true, the private key will be persisted to the iroh data root.
```

Comment:
This was just a placeholder until we got the config working, now that this adds proper config, we should probably just remove this option and always persist it to iroh data root, like we did in the previous version of iroh.

Reply:
So key is always read from iroh home? Fine by me, but that will complicate one shot usage, see below.